### DISCUSSION TITLE
Add Favorites Panel

### DISCUSSION THREAD

**Discussion by warrendugan** [nrwl/nx-console#1025] [Feature Requests]
There are already multiple accordion sections (let's call them panels) in Nx Console (i.e. UI, NX Affected, Workspace JSON, etc.). I would like the ability to favorite commands and have these commands immediately available in this new panel as well as in the command palette in VS Code.

**Comment by asherbig** [comment]
This would make the console so much more usable, especially for people less familiar with nx. Right now if I tell our org to go into the console and use the generators, they get overwhelmed by the options, and often times there's multiple generators that pop up when they search for a specific thing. If there were favorited generators, that would make it less daunting, and help people know they're using the right one for the workspace.

**Reply by Cammisuli** [reply]
If this is something that you'd expect to share with an org, do you expect this to be exported/imported somehow? Or read a config file from a directory?

**Reply by asherbig** [reply]
Hmmm, maybe a list configured in the workspace.json, but also give people the option to edit a local favorites list that doesn't get pushed to source control. The exact setup could be different, but yeah, some easy way to share a config at a workspace level would be needed

**Comment by kyleconkright** [comment]
I would get a lot of use out of this as well. I'd be thrilled if I could favorite a project and have a new panel or stick it to the top of the list.

**Comment by MaxKless** [comment]
We just released a feature to make the common commands configurable... do you think that could help with your usecase?

**Reply by kyleconkright** [reply]
I'll check it out!
